libr/bin/format/dmp/dmp64.c

```diff
@@ -67,6 +67,7 @@ static int r_bin_dmp64_init_header(struct r_bin_dmp64_obj_t *obj) {
 	return true;
 }
 
+#if 0
```

Comment:
same

Reply:
No, not in this case. As far I understand this code might be useful in the future, for implementing more features: https://github.com/radareorg/radare2/issues/16048